packages/ui/src/server/routes.ts

```diff
@@ -55,7 +55,7 @@ const routes: AppRoutes = {
     'GET /api/yt-proxy/:ytAuthCluster/:command': {handler: ytProxyApi},
 
     'GET /api/odin/proxy/:action/:ytAuthCluster?': {handler: odinProxyApi},
-    'GET /api/odin/clusters/availability': {handler: getClustersAvailability},
+    'GET /api/odin/clusters/availability/:ytAuthCluster?': {handler: getClustersAvailability},
```

Comment:
Any changes in `routes` require corresponding changes in `src/ui`. It looks like a mistake. Do we really need it?